MinPlatformPkg/Test/TestPointStubDxe/TestPointStubDxeMm.c

```diff
@@ -241,6 +242,117 @@ PublishMmTestPoint (
   }
 }
 
+/**
+  Notification function of END_OF_DXE event group.
+
+  This is a notification function registered on END_OF_DXE event group.
+  When End of DXE is signalled we get the size of the PiSmmCommunicationRegionTable
+  to allocate a runtime buffer used for communicating the MM Testpoint results.
+  This requires the allocated pages to be unblocked for MM which must occur before
+  ReadyToLock.
+
+  @param[in] Event        Event whose notification function is being invoked.
+  @param[in] Context      Pointer to the notification function's context.
+
+**/
+VOID
+EFIAPI
+OnEndOfDxe (
+  IN EFI_EVENT  Event,
+  IN VOID       *Context
+  )
+{
+  EFI_STATUS                                          Status;
+  UINTN                                               CommSize;
+  EFI_MM_COMMUNICATION_PROTOCOL                      *MmCommunication;
+  UINTN                                               MinimalSizeNeeded;
+  EDKII_PI_SMM_COMMUNICATION_REGION_TABLE             *PiSmmCommunicationRegionTable;
+  UINT32                                              Index;
+  EFI_MEMORY_DESCRIPTOR                               *Entry;
+  UINTN                                               Size;
+

```

Comment:
It looks like this event is never closed. Are you expecting this to be triggered multiple times, or is one time enough (and it should be closed)?

Reply:
I removed this event as it was no longer needed.